deploy/destroy.sh

```diff
@@ -6,6 +6,10 @@ if [[ "$HOSTAPI" == "" ]]; then
   exit 1
 fi
 
+if [[ "$CLEANUP" == "" ]]; then
+  CLEANUP="false"
+fi
+
 pushd $_dir/deploy
-ansible-playbook -e mig_ui_cluster_api_endpoint="$HOSTAPI" -e state=absent playbook.yaml
+ansible-playbook -e mig_ui_cluster_api_endpoint="$HOSTAPI" -e state=absent playbook.yaml --extra-vars='{"mig_ui_remove_namespace": '$CLEANUP'}' 
```

Comment:
You should use the same pattern as the other `-e` vars here.

Reply:
the problem with using -e is that all variables passed with -e are parsed as strings. while the CLEANUP is a boolean. perhaps, you could suggest a better way to pass boolean value